addonscript/src/addonscript/utils.py

```diff
@@ -75,7 +75,8 @@ async def amo_put(context, url, data):
         async with resp as r:
             log.debug('amo_put() for URL "{}" returned HTTP status code: {}'.format(url, r.status))
             r.raise_for_status()
-            returned_value = await r.json()
+            # we silence aiohttp in case we have Null returns from AMO API
+            returned_value = await r.json(content_type=None)
```

Comment:
To silence errors like for None returns from AMO API. Not sure if there's a better workaround this.

```
    raise ContentTypeError(
aiohttp.client_exceptions.ContentTypeError: 0, message='Attempt to decode JSON with unexpected mimetype: ', url=URL('https://addons.allizom.org/api/v4/applications/firefox/82.0/')
exit code: 1
```